storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -3165,6 +3163,16 @@ public void submitTopologyWithOpts(String topoName, String uploadedJarLocation,
                 topoConf.remove(Config.TOPOLOGY_CLASSPATH_BEGINNING);
             }
 
+            // storm.messaging.netty.authentication is about inter-worker communication
```

Comment:
I feel https://github.com/apache/storm/blob/master/storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java#L1098 is a better place for this.